products/sle/main.pm

```diff
@@ -458,6 +458,9 @@ sub load_inst_tests() {
             loadtest 'installation/partitioning_full_lvm';
         }
         if (get_var("FILESYSTEM")) {
+            if (get_var('PARTITIONING_WARNINGS')) {
+                loadtest 'installation/partitioning_warnings';
+            }
```

Comment:
I see. I think because these test steps should be explicit I suggest to add a testsuite to Tumbleweed, @DimStar77 ?